lib/node_modules/@stdlib/math/base/special/frexp/lib/assign.js

```diff
@@ -106,10 +108,10 @@ function frexp( out, x ) {
 	high |= SET_EXP_MASK;
 
 	// Create a new floating-point number:
-	x = fromWords( high, WORDS[ 1 ] );
+	x = fromWords( high, WORDS[ offset + stride ] );
```

Comment:
```suggestion
	x = fromWords( high, WORDS[ 1 ] );
```

Workspace is distinct from the output array.